app/components/PaginatedDropdown.tsx

```diff
@@ -0,0 +1,181 @@
+import * as React from "react";
+import { useMenuState, MenuButton } from "reakit/Menu";
+import styled from "styled-components";
+import User from "~/models/User";
+import Button, { Inner } from "~/components/Button";
+import ContextMenu from "~/components/ContextMenu";
+import MenuItem from "~/components/ContextMenu/MenuItem";
+import Text from "~/components/Text";
+import InputSearch from "./InputSearch";
+import PaginatedList, { PaginatedItem } from "./PaginatedList";
+
+type TFilterOption = PaginatedItem & {
+  id: string;
+  label: string;
+  note?: string;
+};
+
+type Props = {
+  users: any;
```

Comment:
Merging `PaginatedDropdown` into `FilterOptions` would require a type intersection, like this

``` jsx
  12 type TFilterOption = PaginatedItem & {
  13   key: string;
  14   label: string;
  15   note?: string;
  16 };
  17
```

because 

https://github.com/outline/outline/blob/66dbcde29ba1a5a3e95f4e9938507a9ac1df66fa/app/components/PaginatedList.tsx#L42

Although refactoring should be easy enough.

Does that seem alright?